hail/python/hail/expr/aggregators/aggregators.py

```diff
@@ -900,14 +900,21 @@ def hardy_weinberg_test(expr) -> StructExpression:
     :class:`.StructExpression`
         Struct expression with fields `het_freq_hwe` and `p_value`.
     """
+    if getattr(hardy_weinberg_test, '_as_scan', False):
```

Comment:
can you add a test to ensure that this works, specifically? Adding a test for hardy_weinberg scans would probably do that adequately.

Reply:
added test